rules/recognizing-phishing-urls/rule.md

```diff
@@ -26,40 +26,41 @@ For the address <https://www.ssw.com.au/services>, services is the path. The pat
 
 You should always check that the **domain** matches the service or website you are expecting.
 
-<http://linkedin.com.sggr.ru/someaddress>
-
-❌Bad example – the address has LinkedIn in it, but it is a sub-domain, not the domain
-
- 
-
-<http://linked-in-hq.com/linkedin/myprofile>
-
-❌Bad example – the address has LinkedIn in it, but it is in the path, not the FQDN. The FQDN is also suspicious.
-
- 
-
-[http://linkedinalerter.com](http://anzalertercom/)
-
-❌Bad example – the address has LinkedIn in it, but is not a legitimate LinkedIn site.
-
- 
-
- 
-
-<http://linkedin.com/someaddress>
+::: greybox
+<div>http://linkedin.com.sggr.ru/someaddress</div>
+:::
+::: bad
+Bad Example – the address has LinkedIn in it, but it is a sub-domain, not the domain
+:::
 
-✔Good example – LinkedIn is the domain
 
+::: greybox
+<div>http://linked-in-hq.com/linkedin/myprofile</div>
+:::
+::: bad
+Bad Example – the address has LinkedIn in it, but it is in the path, not the FQDN. The FQDN is also suspicious
+:::
  
+::: greybox
+<div>http://linkedinalerter.com</div>
+:::
+::: bad
+Bad Example – the address has LinkedIn in it, but is not a legitimate LinkedIn site
+:::
 
-If you are curious about a URL, and think it might be legitimate, you can check the Whois record to see who owns the domain. A useful tool for this is <https://whois.domaintools.com>
+::: greybox
+<div>http://linkedin.com/someaddress</div>
```

Comment:
1. Instead of <div>, add &#46; as the dots on URLs... that will avoid being clickable
2. In the good example, make it HTTPS

Please action